universal-login-commons/lib/core/utils/sanitizers/asBigNumber.ts

```diff
@@ -0,0 +1,13 @@
+import {utils} from 'ethers';
+import {Sanitizer, Result} from '@restless/sanitizers';
+
+export const asBigNumber: Sanitizer<utils.BigNumber> = (value, path) => {
```

Comment:
Already exist in restless:
https://github.com/EthWorks/restless/tree/master/ethereum#asbignumber

Reply:
It doesn't work. I will create PR for restless in the weekend